src/main/lens-proxy.ts

```diff
@@ -40,6 +40,24 @@ export interface LensProxyFunctions {
   kubeApiRequest: (args: ProxyApiRequestArgs) => void | Promise<void>;
 }
 
+const truthyParams = ["1", "true", ""];
+const watchParam = "watch";
+const followParam = "follow";
+
+export function isLongRunningRequest(reqUrl: string) {
+  const url = new URL(reqUrl, "http://localhost");
+
+  if (url.searchParams.has(watchParam) && truthyParams.includes(url.searchParams.get(watchParam)) ) {
```

Comment:
I would use the `getBoolean` method from `src/main/routes/utils/` (and maybe move it) unless you feel that this shouldn't prioritize false.

Reply:
I didn't know that function.... but that function returns false if value is empty. Is that expected behaviour also here 🤔

Reply:
No it doesn't. The function `get` return `null` if the key is not set

Reply:
Right, refactored to use `getBoolean`.